llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/Input/InputOps.td

```diff
@@ -201,9 +203,61 @@ def IREEInput_GlobalStoreIndirectOp : IREEInput_Op<"global.store.indirect"> {
 }
 
 //===----------------------------------------------------------------------===//
-// Buffer Views
+// Buffers and Buffer Views
 //===----------------------------------------------------------------------===//
 
+def IREEInput_BufferViewCreateOp : IREEInput_PureOp<"buffer_view.create"> {
+  let summary = [{buffer view reference initializer}];
+  let description = [{
+    Creates a reference to a buffer with a particular shape and element type.
+    The buffer is not copied and both the original and view references must be
+    synchronized. This makes it easier to associate commonly-carried metadata
+    along with the contents.
+  }];
+
+  let arguments = (ins
```

Comment:
Do these match the flow ones exactly and is the intention to keep them close while this is the more stable one?

Reply:
Right now it's exact match, but in future I think they might diverge a little bit